src/api/mapping.ts

```diff
@@ -0,0 +1,17 @@
+import express from 'express'
+import uuid4 from 'uuid/v4'
+const mappingRouter = express.Router()
+const list =[]
+
+mappingRouter.post('/mappings', (req, res) =>{
```

Comment:
Look at comment above. If you are using `/mappings` up above, then this should be `post('/', (req, res)....`